.github/workflows/openSSFscore.yml

```diff
@@ -0,0 +1,58 @@
+name: Scorecard analysis workflow
+on:
+  # Only the default branch is supported.
+  branch_protection_rule:
+  schedule:
+    # Weekly on Saturdays.
+    - cron: '30 1 * * 6'
+  pull_request:
+  push:
+    branches: [ main ]
+
+# Declare default permissions as read only.
+permissions: read-all
+
+jobs:
+  analysis:
+    name: Scorecard analysis
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed if using Code scanning alerts
+      security-events: write
+      # Needed for GitHub OIDC token if publish_results is true
+      id-token: write
+
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v3.5.2
```

Comment:
@rootfs  I suppose we don't have time/effort for now as tracing specific hash id.... or per pr commit just update hash id.... Hence I suppose it's too early for us to enable every check items.

Reply:
Repeat myself, before we have a clearance on the value:
- as which openSSF standard is consume by whom 
- how they consume it
I am open for openSSF standard, but basing on our own effort to support it.
if following an standard means too many effort as follow up commit hash, without a clearance on value... I am doubting myself as why should I make it?

Reply:
We don't need to fix everything at once. But this might be interesting to fix.

Using the commit hash instead of version:
```
This protects us from supply chain attacks against this repository as 
only GitHub (who already has the ability to completely screw us) could 
sneak in a malicious change to a dependency, which has privileged access.
Source: https://github.com/ethereum/EIPs/issues/5298
```

For example, instead of using:
```
uses: actions/checkout@v3.5.2
```
We should use:
```
uses: actions/checkout@98637d7db679cd12b4ae373378e391551b922539
```
Where the hash is from the gihub commit: https://github.com/actions/checkout/pull/1376/commits

Reply:
@marceloamaral , if you further step into https://github.com/sustainable-computing-io/kepler/security/code-scanning/33
1. we are not nodejs
2. we are not docker
3. we are not building...
ok... I just use it to generate a scan report? and which is out of the scope of the issue.
hence I suppose it's false positive.

Reply:
Can you try to use the hash to see if it fix the issue?